static_src/components/app_container.jsx

```diff
@@ -60,12 +61,14 @@ export default class AppContainer extends React.Component {
     AppStore.addChangeListener(this._onChange);
     OrgStore.addChangeListener(this._onChange);
     SpaceStore.addChangeListener(this._onChange);
+    pageMetadataActions.loadPage('app');
```

Comment:
So rather then having the components fire this, what if they're fired in the main.js file. Then the title bar would set the current page type to `app` and since it's listening to the stores, it would re-update when the data comes in?